### PR TITLE
Feature/cwls import

### PR DESCRIPTION
**Pull request by davidrajones**

### CODE REVIEW

**Review by keiranmraine — COMMENTED** [review]
It looks fine, provided the CWL has been confirmed working.  @byb121 may already have done this prior to the migration.

**Comment by byb121** [comment]
It was a tiny change that I don't think can break anything, so didn't do any tests on CWLs. I'll have a quick go on this repo.

File: cwls/bam2bedgraph.cwl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+class: CommandLineTool
+
+id: "cgpbigwig-bam2bedgraph"
+label: "cgpbigwig-bam2bedgraph"
+
+baseCommand: bam2bedgraph
+
+requirements:
+  - class: DockerRequirement
+    dockerPull: "quay.io/wtsicgp/cgpBigWig:1.3.0"
```
Comment (byb121) [inline cwls/bam2bedgraph.cwl]: only lower cases are allowed in the registry address. Same place in other CWLs need to be corrected as well.

File: cwls/bam2bedgraph.cwl [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+s:author:
+  - class: s:Person
+    s:identifier: https://orcid.org/0000-0002-0407-0386
+    s:email: mailto:drj@sanger.ac.uk
+    s:name: David Jones
+
+dct:creator:
+"@id": "https://orcid.org/0000-0002-0407-0386"
+foaf:name: David Jones
+foaf:mbox: "drj@sanger.ac.uk"
```
Comment (byb121) [inline cwls/bam2bedgraph.cwl]: schema.org works fine, so I think we can remove these lines. `foaf` lines create warnings when running with dockstore.

File: cwls/bam2bedgraph.cwl [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      prefix: --filter
+  overlap:
+    type: boolean
+    doc: "Flag to check for overlapping reads"
+    inputBinding:
+      prefix: --overlap
+
+outputs:
+  output:
+    type: stdout
```
Comment (byb121) [inline cwls/bam2bedgraph.cwl:56]: out of my curiosity, why not output to a file?

File: cwls/bam2bedgraph.cwl (hunk 1 above)
Comment (davidrajones) [reply cwls/bam2bedgraph.cwl]: 1b1e882

File: cwls/bam2bedgraph.cwl (hunk 2 above)
Comment (davidrajones) [reply cwls/bam2bedgraph.cwl]: 1b1e882

File: cwls/bam2bedgraph.cwl (hunk 3 above)
Comment (davidrajones) [reply cwls/bam2bedgraph.cwl:56]: A default of stdout allows you to then pipe to further processing. However I'm not sure if that helps in the cwl

**Pull request merged**